cache/lru_cache.h

```diff
@@ -150,9 +150,10 @@ class LRUHandleTable {
 
   // The table consists of an array of buckets where each bucket is
   // a linked list of cache entries that hash into the bucket.
-  uint32_t length_;
-  uint32_t elems_;
-  LRUHandle** list_;
+  // cache line alignment to avoid false sharing.
+  uint32_t length_ CACHE_ALIGNED(CACHE_LINE_SIZE);
+  uint32_t elems_ CACHE_ALIGNED(CACHE_LINE_SIZE);
+  LRUHandle** list_ CACHE_ALIGNED(CACHE_LINE_SIZE);
```

Comment:
`length_`  and `list_`  almost never change while `elems_` changes frequently. Can we reorder `length_` and `elems_` so that we don't have to align `list_` to save a little bit memory?